tests/integration/config/plugins/integration-tests/data/tests/tests_illegal_atrocity.txt

```diff
@@ -0,0 +1,240 @@
+			name "Buggy Barge"
+			sprite "ship/star barge"
+			attributes
+				category "Light Freighter"
+				cost 190000
+				mass 10
+				bunks 3
+				"cargo space" 50
+				drag 1
+				"engine capacity" 40
+				"fuel capacity" 300
+				"heat dissipation" 0.8
+				hull 1000
+				"outfit space" 130
+				"required crew" 1
+				shields 600
+				"turret mounts" 1
+				"weapon capacity" 20
+			outfits
+				"X1700 Ion Thruster"
+				"X1200 Ion Steering"
+				Hyperdrive
+				"nGVF-BB Fuel Cell"
+				"LP036a Battery Pack"
+				"Nerve Gas"
+			crew 1
+			fuel 300
+			shields 600
+			hull 1000
+			position 0 0
+			engine -9 38 1
+			engine 9 38 1
+			turret 0 -8
+			leak leak 60 50
+			explode "tiny explosion" 10
+			explode "small explosion" 10
+			system "Terra Incognita"
+			planet Ruin
+		account
+			credits 100000000
+			score 400
+			history
+
+		event "atrocity stuff"
+			government "Quarg"
+				atrocity
+					Hyperdrive
+				# they don't have it yet but to be sure
+				remove "death sentence"
+				remove "enforces"
+		visited "Terra Incognita"
+		"visited planet" Ruin
+		conditions
+			"%TEST%: SCANNING ILLEGALS"
+
+
+ship "Quarg Wardragon" "%TEST%: Dumbdragon"
+	outfits
+		"Antimatter Core"
+		"Medium Graviton Steering"
+		"Medium Graviton Thruster"
+		"Nanotech Battery"
+	add attributes
+		"outfit scan power" 1000000000
+		"outfit scan speed" 1000000000
+
+mission "%TEST%: Wardragon Scanning You!"
+	priority
+	landing
+	to offer
+		has "%TEST%: SCANNING ILLEGALS"
+	on offer
+		conversation
+			"You must be scanned or be killed."
+				accept
+	npc assist
+		government "Quarg"
+		# I used another definition that had 1 million scan power but whatever
+		ship "%TEST%: Dumbdragon" "Scanner"
+		personality derelict surveillance target
+
+test "Land On Ruin helper"
+	status partial
+	description "Helper for landing on Ruin."
+	sequence
+		input
+			command land
+		label landing
+		branch landing
+			"flagship planet: Ruin" < 1
+		assert
+			"flagship planet: Ruin" > 0
+
+test "Illegal Test - Ignore and New Illegal"
+	status active
+	description "Test if a Quarg ship spawning disabled in the system will give you a fine for an illegal outfit and ignore the fine for another, after you repaired it."
+	sequence
+		inject "quarg scanning"
+		call "Load First Savegame"
+		watchdog 12000
+		assert
+			"fines" == 0
+		call "Depart"
+		# land again to trigger the mission
+		call "Land On Ruin helper"
+		input
+			command "enter"
+		call "Depart"
+		# board the ship so we're sure we get close to it to get scanned
+		input
+			command board
+		# Wait for 10 seconds so it should be repaired
+		apply
+			"test: steps to wait" = 10 * 60
+		label "floating"
+		# empty input to make time pass
+		input
+		apply
+			"test: steps to wait" = "test: steps to wait" - 1
+		branch "floating"
+			"test: steps to wait" > 0
+		# skip the dialog message for the fine
+		input
+			command "enter"
+		# land back with a fine of 1
+		call "Land On Ruin helper"
+		assert
+			"fines" == 1
```

Comment:
Is this an existing automatic variable? If so, then I guess this test is nice. (And otherwise we might want to check if "unpaid fines" is above zero, that condition exists.)

Reply:
Right this one isnt the right one